snap/snapcraft.yaml

```diff
@@ -24,21 +24,24 @@ parts:
     source: https://gitlab.gnome.org/jwestman/blueprint-compiler.git
     source-tag: 'v0.18.0'
     source-depth: 1
+    build-attributes: [enable-patchelf]
     override-pull: |
       craftctl default
-      craftctl set version=$(git describe --abbrev=0 --tags | cut -c 2-)
+      craftctl set version=$(git describe --abbrev=0 --tags)
     meson-parameters:
       - --prefix=/snap/blueprint-compiler/current/usr
     organize:
       snap/blueprint-compiler/current: .
     build-packages:
       - meson
+    stage-packages:
+      - python3-gi
     prime:
       - -snap/blueprint-compiler
 
 apps:
   blueprint-compiler:
     command: usr/bin/blueprint-compiler
     environment:
-      GI_TYPELIB_PATH: /snap/gnome-46-2404-sdk/current/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/girepository-1.0:/snap/gnome-46-2404-sdk/current/usr/lib/girepository-1.0:/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/girepository-1.0:/usr/lib/girepository-1.0:$GI_TYPELIB_PATH
-      PYTHONPATH: $SNAP/usr/lib/python3/dist-packages:$PYTHONPATH
+      GI_TYPELIB_PATH: $SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/girepository-1.0:$SNAP/usr/lib/girepository-1.0:/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/girepository-1.0:/usr/lib/girepository-1.0${GI_TYPELIB_PATH:+:$GI_TYPELIB_PATH}
```

Comment:
The snap doesn't contain any gi typelib defintions. This binary is probably never invoked with the `GI_TYPELIB_PATH` variable, and also doesn't take up env variables from the environment, so I think this won't work inside an lxd environment. That's why I specifically added those paths. Did it work this time?